### ISSUE TITLE
Typo in OAuth.rst

### ISSUE THREAD

**Issue by MasatoshiTada** [terasolunaorg/guideline#2795]
## Description

I found some typo in OAuth.rst.

### 9.9.2.4.2. リソースにアクセス可能なスコープの設定

http://terasolunaorg.github.io/guideline/5.3.0.RELEASE/ja/Security/OAuth.html#id28

- Wrong
```xml
<sec:custom-filter ref="oauth2ProviderFilter"
                        before="PRE_AUTH_FILTER" />
```

- Correct
```xml
<sec:custom-filter ref="oauth2AuthenticationFilter"
                        before="PRE_AUTH_FILTER" />
```

### 9.9.2.5.1.2. OAuth2ClientContextFilterの適用 (Only Japanese version)

http://terasolunaorg.github.io/guideline/5.3.0.RELEASE/ja/Security/OAuth.html#oauth2clientcontextfilter

- Wrong
`spring-mvc.xm`

- Correct
`spring-mvc.xml`

### 9.9.2.5.2. JavaScriptを使用したリソースへのアクセス

http://terasolunaorg.github.io/guideline/5.3.0.RELEASE/ja/Security/OAuth.html#javascript

- Wrong

```jsp
<script type="text/javascript">
"use strict";

$(document).ready(function() {
  // omitted
}; // ")" is missing

</script>
```

- Correct

```jsp
<script type="text/javascript">
"use strict";

$(document).ready(function() {
  // omitted
});

</script>
```

## Possible Solutions
See above.

## Affects Version/s
* 5.3.0.RELEASE

## Fix Version/s
(To be written later by project member)
- [ ] 5.4.0_ja (master)
- [ ] 5.3.1_ja (5.3.x)
- [ ] 5.4.0_en (master)
- [ ] 5.3.1_en (5.3.x)

## Change log(should be updated at the same time)
(To be selected later by project member)
- [ ] Required
- Not Required

## English Translation
(To be selected later by project member)
- [ ] Required, handle with the same issue
- [ ] Required, create a new issue for handling
- Not Required 

## Migrating Guide
(To be selected later by project member)
- [ ] Required
- Not Required

## Issue Links
- http://terasolunaorg.github.io/guideline/5.3.0.RELEASE/ja/Security/OAuth.html

**Comment by itouyun** [comment]
@MasatoshiTada
Thank you for your finding some typo.

**Comment by itouyun** [comment]
@MasatoshiTada
After confirming it with the development team, the issue was already known as a clerical error.
The chapter of OAuth is still under improvement now, and correcting this issue in also in our plan.

The above mentioned correction is scheduled to be released in 2017 Summer or 2018 Winter.

Please allow me to close this Pull Request. 
And this issue will be merged into another issue later on.

**Comment by MasatoshiTada** [comment]
OK, I understand. Thank you for your kindness.
I look forward to next release!

**Comment by btkigamim** [comment]
@MasatoshiTada
Thank you for your comment.
I will respond to this issue on issue #2891.

**Comment by MasatoshiTada** [comment]
Great! Thank you for your kind correspondence.

**Comment by NestorSancho** [comment]
Hi!
There is another typo in the source code block of the section "9.10.2.5.1.3.3. Settings of resource at the time of using client credential grant":

- Wrong:

<oauth2:rest-template **resource=id**="todoClientGrantResourceRestTemplate" resource="todoClientGrantResource" />

- Right:

<oauth2:rest-template **id**="todoClientGrantResourceRestTemplate" resource="todoClientGrantResource" />

**Comment by btkigamim** [comment]
@NestorSancho 
I am sorry for the delay in my reply.

Thank you for your finding typo.
I will also respond to this issue with issue #2891.

**Comment by itouyun** [comment]
This issue bug is already fixed by #2891, this issue is closed.
Thank you for your comments!
